docs/gettingstarted/index.rst

```diff
@@ -94,12 +94,9 @@ Getting started with Flocker
 
          Install the Flocker client on your Linux machine:
 
-         .. version-code-block:: console
+         .. task:: install_cli ubuntu-14.04
```

Comment:
The old install instructions were (flocker) version specific, these aren't.

There is now inconsistency in the presentation of these instructions (e.g. $ prompt) and the other ones in this guide (e.g. you@laptop:~$ ).